styles/_govuk-elements.scss

```diff
@@ -0,0 +1,30 @@
+// GOV.UK elements
+@import "govuk-elements-sass/public/sass/elements/helpers";                       // Helper functions and classes
```

Comment:
You can import this file: https://github.com/alphagov/govuk_elements/blob/master/packages/govuk-elements-sass/public/sass/_elements.scss